rubicon_ml/cli.py

```diff
@@ -78,7 +80,78 @@ def ui(root_dir, page_size, project_name, host, port, debug, storage_options):
     dashboard.serve(run_server_kwargs=run_server_kwargs)
 
 
-# CLI groups
+@cli.command(
+    context_settings=dict(
+        ignore_unknown_options=True,
+    )
+)
+@click.option(
+    "--root-dir",
+    type=click.STRING,
+    default=os.environ.get("RUBICON_ROOT_DIR"),
+    help="The absolute path to the top level folder holding the rubicon-ml project.",
+    required=False,
+)
+@click.option(
+    "--project-name",
+    default=os.environ.get("RUBICON_PROJECT_NAME"),
+    type=click.STRING,
+    help="Name of rubicon project to retrieve experiments from",
+    required=False,
+)
+@click.option(
+    "--color",
+    "-c",
+    type=click.STRING,
+    help="Color of query output",
+    required=False,
+)
+@click.argument(
+    "query",
+    nargs=1,
+    required=True,
+)
+def search(root_dir, project_name, color, query):
+    """
+    This function provides capability to query  rubicon experiments from the command line without using python
+    directly and output the JSON in the command line.
+    """
+
+    proj_name_env = "RUBICON_PROJECT_NAME"
+    root_dir_env = "RUBICON_ROOT_DIR"
+    fg_color = "bright_yellow"
+
+    if (proj_name_env not in os.environ and project_name is None) or (
+        root_dir_env not in os.environ and root_dir is None
+    ):
+        warnings.warn(
+            "No previous project name and/or Rubicon root directory saved, enter a new one"
+        )
+        return
 
+    if not project_name:
+        project_name = os.environ["RUBICON_PROJECT_NAME"]
+    if not root_dir:
+        root_dir = os.environ["RUBICON_ROOT_DIR"]
```

Comment:
pretty sure this block is redundant as well. click will set `project_name` equal to `os.environ.get("RUBICON_PROJECT_NAME")` via the default. same with the other

also if either of these are `None` we'd never get to this point since the block above this would raise the warning and return